odl/solvers/nonsmooth/douglas_rachford.py

```diff
@@ -28,7 +28,7 @@
 __all__ = ('douglas_rachford_pd',)
 
 
-def douglas_rachford_pd(x, prox_f, prox_cc_g, L, tau, sigma, niter,
+def douglas_rachford_pd(x, f, g, L, tau, sigma, niter,
```

Comment:
I have also grown to like the very short `f`, `g` etc mostly because frankly any name we come up with wont mean much unless you have read the documentation, and these can at least easily be tied with the research article.

The largest change I'd do is to make something imply that the g's can be a list (frankly we should add some code that relaxes this assumption and allows single functions as input.) Perhaps `gi`, `gs` etc.

Regarding `x` this is in following with the convention of `x0` used in scipy. Anything longer would clog all the interfaces with minimal gain, especially since this is almost universally called `x` in the optimization literature.